config/sync/field.storage.node.panelizer.yml

```diff
@@ -1,3 +1,4 @@
+uuid: 01873d03-f1fb-4c26-b712-fc7ee11938f6
```

Comment:
due to this task, this file should not be overwritten:
https://app.scrumdo.com/projects/story_permalink/1437160